components/common/link/NavigationLink.tsx

```diff
@@ -5,9 +5,9 @@ import { useRouter } from 'next/router';
 
 import MuiLink, { LinkProps } from '@mui/material/Link';
 
-type NavigationLinkProps = LinkProps & NextLinkProps;
+type NavLinkProps = LinkProps & NextLinkProps;
```

Comment:
The main rule of thumb is to follow `<ComponentName>Props` pattern in naming interfaces and types.

Reply:
'+